apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,108 +18,176 @@ export function InstructorCourseAdminSettings({
       </head>
       <body>
         ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
-        <main id="content" class="container-fluid">
+        <main id="content" class="container mb-5 px-0">
           ${renderEjs(__filename, "<%- include('../partials/courseSyncErrorsAndWarnings'); %>", {
             ...resLocals,
           })}
-          <div class="card mb-4">
-            <div class="card-header bg-primary text-white d-flex">Settings</div>
-            <table class="table table-sm table-hover two-column-description">
-              <tbody>
-                <tr>
-                  <th>Short Name</th>
-                  <td>${resLocals.course.short_name}</td>
-                </tr>
-                <tr>
-                  <th>Title</th>
-                  <td>${resLocals.course.title}</td>
-                </tr>
-                <tr>
-                  <th>Institution</th>
-                  <td>${resLocals.institution.short_name} (${resLocals.institution.long_name})</td>
-                </tr>
-                <tr>
-                  <th>Timezone</th>
-                  <td>${resLocals.course.display_timezone}</td>
-                </tr>
-                <tr>
-                  <th>Path</th>
-                  <td>${resLocals.course.path}</td>
-                </tr>
-                <tr>
-                  <th>Repository</th>
-                  <td>${resLocals.course.repository}</td>
-                </tr>
-                <tr>
-                  <th>Configuration</th>
-                  <td>
-                    ${!coursePathExists
-                      ? resLocals.authz_data.has_course_permission_edit &&
+          <div class="card">
+            <div class="card-header bg-primary text-white d-flex mb-2">Course Settings</div>
+            <form class="mx-4">
```

Comment:
Rather than adding `mb-2` to the header and `mx-4` to the form, you can use the `card-body` class on the `form`: https://getbootstrap.com/docs/4.6/components/card/#body

Reply:
Updated to use `card-body` in the form in [fdd54a4](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/fdd54a49777af842be8173face2752996c726671).